_posts/2025-04-28-sparse-autodiff.md

```diff
@@ -8,7 +8,17 @@ htmlwidgets: true
 hidden: false
 
 authors:
-  - name: Anonymous
+  - name: Adrian Hill
+    url: "https://adrianhill.de"
+    affiliations:
+      name: BIFOLD, Technical University of Berlin
+  - name: Guillaume Dalle
+    url: "https://gdalle.github.io/"
+    affiliations:
+      name: LVMT, ENPC, Institut Polytechnique de Paris, Univ Gustave Eiffel, Marne-la-Vallée, France
```

Comment:
Maybe you skipped the issue text, but the given example and sample blogpost suggest much simpler formatting of affiliations:
<img width="769" alt="image" src="https://github.com/user-attachments/assets/5d5ba4fa-941e-4aa2-8bb9-c9bdd875cab4" />
<img width="591" alt="image" src="https://github.com/user-attachments/assets/f5234fe1-6cbd-4b38-8403-a6e639f70ad3" />

Reply:
I don't have a choice on the formatting of my affiliation.

Reply:
Alright, then I'll update mine as well. The issue is that the template doesn't seem to support multiple affiliations, so I would have to put 

*BIFOLD – Berlin Institute for the Foundations of Learning and Data, Berlin, Germany, Machine Learning Group, Technical University of Berlin, Berlin, Germany*

to match your style.

Reply:
This looks very silly if you ask me:
<img width="620" alt="image" src="https://github.com/user-attachments/assets/c2838905-b0c8-489e-ae38-729eb73dfbb9" />

Reply:
Alright, lets roll with it then

Reply:
For information, "Argonne National Laboratory" is fine for me.